addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.scss

```diff
@@ -1,5 +1,34 @@
+.payment-summary {
+  padding-inline: 150px;
+}
+
 .payment-method-icon {
     max-width: 50px;
     max-height: 50px;
     vertical-align: middle;
 }
+
+.paymentmethods-small {
+  max-height: 186px;
+}
+
+.left-content .numpad button {
+    font-size: 1.1rem;
+}
```

Comment:
can you have a way to not need this custom css? Preferably with bootstrap

Reply:
There doesn't appear to be a Bootstrap class for the desired font size. Additionally, this font size adjustment needs to be applied exclusively to the numpad buttons on the payment screen, without affecting other screens.

Reply:
> There doesn't appear to be a Bootstrap class for the desired font size.

I am sure that among all the bootstrap utilities you will find what you need.

> Additionally, this font size adjustment needs to be applied exclusively to the numpad buttons on the payment screen, without affecting other screens

You can give a custom class to the buttons. [Reference](https://github.com/odoo/odoo/blob/master/addons/point_of_sale/static/src/app/generic_components/numpad/numpad.js#L13)